Spokestack/WebRTCVAD.swift

```diff
@@ -55,7 +55,6 @@ private var frameBuffer: RingBuffer<Int16>!
     
     deinit {
         vad.deinitialize(count: 1)
-        WebRtcVad_Free(vad.pointee)
```

Comment:
This change was intentional, right? Just checking; I know I've asked about these calls before.

Reply:
yeah, this is the "fix double free bug in webtrcvad" part of the pr. found another condition (caused by the line change you called out above) that causes a double free.